### PR TITLE
Correct pred:exp pKa pairining error in type III pKa analysis.

### PR DESCRIPTION
**Pull request by MehtapIsik**
There was an error in analysis script that caused mispairing of some experimental and predicted pKas, which caused higher than expected RMSE values. Analysis script is now fixed and figures updated.

@bannanc @davidlmobley @jchodera Could you please review?

### CODE REVIEW

**Comment by MehtapIsik** [comment]
Link to README for this analysis: https://github.com/MobleyLab/SAMPL6/blob/pKa_typeIII_analysis/physical_properties/pKa/analysis/analysis_of_typeIII_predictions/README.md

File: physical_properties/pKa/analysis/analysis_of_typeIII_predictions/typeIII_analysis.py [hunk 1]
```diff
@@ -66,6 +66,8 @@ def compute_bootstrap_statistics(samples, stats_funcs, percentile=0.95, n_bootst
         stats_funcs = [stats_funcs]
 
     # Compute mean statistics.
+    #print("\nSamples:\n", samples)
+
```
Comment (bannanc) [inline physical_properties/pKa/analysis/analysis_of_typeIII_predictions/typeIII_analysis.py]: @MehtapIsik Is there a reason to have these print statements in the code, but commented out. I would say you should either include them or delete the lines.

**Comment by davidlmobley** [comment]
@bannanc - take a look? I'm about to go offline for the next few hours so I'll mark as "approved" in case you guys decide it is good to merge; I certainly won't have time to look today.

**Comment by bannanc** [comment]
@MehtapIsik and @davidlmobley 
I suggested removing lines in the code that are commented out print statements, but otherwise this is fine to go. I don't have authorization to actually merge it.

File: physical_properties/pKa/analysis/analysis_of_typeIII_predictions/typeIII_analysis.py (hunk 1 above)
Comment (MehtapIsik) [reply physical_properties/pKa/analysis/analysis_of_typeIII_predictions/typeIII_analysis.py]: I'll do a little bit of cleaning.

**Comment by MehtapIsik** [comment]
No problem. It seems like I can merge it. But I will clean up those print statements from the code first.

**Comment by bannanc** [comment]
I think its fine to merge it. It must be set so only admins can approve it. I think you should go ahead and merge whenever you're ready and send an e-mail out to the list so people know the statistics are improved. It might make people more likely to submit an abstract.

**Pull request merged**